gpm_api/tests/io/test_checks.py

```diff
@@ -0,0 +1,478 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 17 15:41:14 2023
+
+@author: ghiggi
+"""
+
+import pytest
+import datetime
+import numpy as np
+from gpm_api.io import checks
+from gpm_api.io.products import available_products, available_scan_modes
+
+
+def test_is_not_empty() -> None:
+    ''' Test is_not_empty() '''
+
+    # Test a non empty object
+    res = checks.is_not_empty([1, 2, 3])
+    assert res is True, "Function returned False, expected True"
+
+    # Test an empty object
+    res = checks.is_not_empty([])
+    assert res is False, "Function returned True, expected False"
+
+
+def test_is_empty() -> None:
+    ''' Test is_empty() '''
+
+    # Test a non empty object
+    res = checks.is_empty([1, 2, 3])
```

Comment:
Same as above comment

Reply:
Also the same as my comment above, if given, `True` for example, the function will return `False`. Is that the expectation?